### ISSUE TITLE
Documentation is in correct 

### ISSUE THREAD

**Issue by krownoss** [gr2m/hapi-cors-headers#7]
```js
server.ext('onPreResponse',addcorsHeaders);
```

line should be

```js
server.ext('onPreResponse',corsHeaders);
```

**Comment by krownoss** [comment]
This is on NPM website where the incorrect documentation is.

**Comment by gr2m** [comment]
Thank you, I’ll release a new version to fix the documentation on npm 👍

**Comment by gr2m** [comment]
fixed with [v1.0.1](https://github.com/gr2m/hapi-cors-headers/releases/tag/v1.0.1)
